data/coalition outfits.txt

```diff
@@ -143,6 +143,21 @@ outfit "Cooling Module"
 	"cooling energy" 1.4
 	description "This cooling module consumes energy. To conserve power, it does not ramp up to full strength unless a ship is very close to overheating."
 
+outfit "Scanning Module"
+	category "Systems"
+	licenses
+		Heliarch
+	cost 215000
```

Comment:
Given that this is a three-in-one scanner, I'd up this cost to about 400k instead of nearly being the same cost as the same power's worth in human scanners (which is roughly 160k; the about 400k cost takes into account the fact that this scanner is about 2.6 times more compact).

Reply:
Actually I don't remember how I decided on its price three months ago. Looking at it again, a single scanning module (mass 3) is about as effective as three cargo scanners, one outfit scanner, and one tactical scanner (mass 3*1+2+4=9), which cost 3*8+24+144=192 thousand. On the other hand, it is not for sale, you can only acquire it by plundering Heliarch ships, so the cost does not really matter. Coalition outfits are typically cheap, though.
Anyway, I've now raised its cost above 0.4 M, hopefully that addresses your concern.